### ISSUE TITLE
convox deploy does not output the URL of the ELB

### ISSUE THREAD

**Issue by parijatmishra** [convox/rack#59]
The documentation at https://docs.convox.com/docs/deploying-an-application/ says that convox deploy will output a release id and then the URL of any processes.  But the last line for me is:

```
$ convox deploy .
Deploying sinatra
Uploading... OK
RUNNING: docker build -t izfznzhvot /tmp/repo967044014/clone
.
.
.
Pushing tag for rev [05127cc0dcbc] on {https://convox-1751103955.us-east-1.elb.amazonaws.com:5000/v1/repositories/sinatra-worker/tags/BBYJIYNLYTJ}
```

I have to use the AWS tools to discover the ELB URL.

I have a docker-compose.yml and Dockerfile. I don't have a Procfile. I am using git clone git@github.com:convox-examples/sinatra.git to test.

Using convox on Mac OSX 10.10

```
$ convox --version
client: 20150922044212
server: 20150919002113 (convox-1751103955.us-east-1.elb.amazonaws.com)
$ uname -a
Darwin <redacted> 14.5.0 Darwin Kernel Version 14.5.0: Wed Jul 29 02:26:53 PDT 2015; root:xnu-2782.40.9~1/RELEASE_X86_64 x86_64
```

**Comment by ddollar** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;EmKt4o2m&#x2F;182-display-app-url-after-deploy"><img src="https:&#x2F;&#x2F;github.trello.services&#x2F;images&#x2F;trello-icon.png" width="12" height="12"> Display app URL after deploy</a>

**Comment by lawrencepit** [comment]
To discover the ELB URL you can also do 

```
$ convox apps info
```

**Comment by csquared** [comment]
this is fixed by #100

**Comment by matti** [comment]
Currently URL is not shown, see:
```
Step 7 : COPY . /app
 ---> 5f93da0f1333
Removing intermediate container 8ef0b854f60c
Successfully built 5f93da0f1333
RUNNING: docker tag -f thhfhhilpn sinatra/web
RUNNING: docker tag -f thhfhhilpn sinatra/worker
RUNNING: docker tag -f sinatra/web 149069325135.dkr.ecr.us-east-1.amazonaws.com/sinatra-afleafizsz:web.BHTXIZPQSVT
RUNNING: docker push 149069325135.dkr.ecr.us-east-1.amazonaws.com/sinatra-afleafizsz:web.BHTXIZPQSVT
The push refers to a repository [149069325135.dkr.ecr.us-east-1.amazonaws.com/sinatra-afleafizsz] (len: 1)
5f93da0f1333: Preparing
5f93da0f1333: Pushing
5f93da0f1333: Pushed
977fe120f7a1: Image already exists
b7a97fa13e10: Image already exists
120cb331b331: Image already exists
82b248dc4370: Image already exists
7163258695a6: Image already exists
d45aca68252b: Image already exists
web.BHTXIZPQSVT: digest: sha256:8810fc44cb42e04d7c647162de87727aeb04e42a7be3c11d7eaaff0c955de151 size: 10709
RUNNING: docker tag -f sinatra/worker 149069325135.dkr.ecr.us-east-1.amazonaws.com/sinatra-afleafizsz:worker.BHTXIZPQSVT
RUNNING: docker push 149069325135.dkr.ecr.us-east-1.amazonaws.com/sinatra-afleafizsz:worker.BHTXIZPQSVT
The push refers to a repository [149069325135.dkr.ecr.us-east-1.amazonaws.com/sinatra-afleafizsz] (len: 1)
5f93da0f1333: Image already exists
977fe120f7a1: Image already exists
b7a97fa13e10: Image already exists
120cb331b331: Image already exists
82b248dc4370: Image already exists
7163258695a6: Image already exists
d45aca68252b: Image already exists
worker.BHTXIZPQSVT: digest: sha256:6516495d7033267f307b583cb13dbbecd56450ff5dcaf7d42600fa1054f63806 size: 10712
Promoting RUQOEQKWYLV... UPDATING
```
